tools/mlir-proto-opt/mlir-proto-opt.cpp

```diff
@@ -54,17 +54,27 @@ static void registerIreeDialects(DialectRegistry &registry) {}
 #endif
 
 void registerTestPasses() { registerTestVectorMaskingUtils(); }
+#ifdef SANDBOX_ENABLE_ALP
```

Comment:
Could you directly add an extra level of indirection please ?

I.e. add `registerExperimentalPasses` and keep the `ifdefs` in that function.

This way future projects only have a single place to modify and the `main` does not grow up over time with mostly optional functions.